<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointOfInterestProperties</class>
 <widget class="QWidget" name="PointOfInterestProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POI Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="mBullseyeCheckbox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="mButtonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mLatLabel">
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mAltLabel">
     <property name="text">
      <string>Altitude</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="UtQtLengthValueEdit" name="mAltitude"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="mName"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mNameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mLonLabel">
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="UtQtLonPosLineEdit" name="mLongitude"/>
   </item>
   <item row="1" column="1">
    <widget class="UtQtLatPosLineEdit" name="mLatitude"/>
   </item>
   <item row="1" column="2">
    <widget class="wkf::LocationSelector" name="mLocationSelector">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mBullseyeLabel">
     <property name="text">
      <string>Bullseye</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wkf::LocationSelector</class>
   <extends>QPushButton</extends>
   <header>WkfSelectorWidgets.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLatPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLonPosLineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitLineEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtLengthValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mName</tabstop>
  <tabstop>mLatitude</tabstop>
  <tabstop>mLocationSelector</tabstop>
  <tabstop>mLongitude</tabstop>
  <tabstop>mAltitude</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
